index.html

```diff
@@ -4,6 +4,12 @@
 		<title>Minecraft Double Smooth Stone Slab</title>
 		<link rel="shortcut icon" type="image/png" href="/assets/favicon.png" />
 		<link rel="icon" type="image/png" href="/assets/favicon.png" />
+		<meta property="og:title" content="Minecraft Double Smooth Stone Slab">
+		<meta property="og:site_name" content="Minecraft Double Smooth Stone Slab">
+		<meta property="og:url" content="https://minecraftdoublesmoothstoneslab.website">
+		<meta property="og:description" content="Minecraft Double Smooth Stone Slab">
+		<meta property="og:type" content="article">
```

Comment:
this property isn't actually required but it should probably just be "website"?
```suggestion
		<meta property="og:type" content="website">
```

Reply:
Possibly. I'll change it and test.

Reply:
Changed to website.